lib/app/utils.js

```diff
@@ -74,7 +74,7 @@ export function flatMapComponents(route, fn) {
 }
 
 export async function resolveRouteComponents(route) {
```

Comment:
So now we can also remove async from `async function` :D

Reply:
oh, that's right.